pcdsdaq/daq.py

```diff
@@ -22,6 +22,10 @@
 # Do not allow begins within this many seconds of a stop
 BEGIN_THROTTLE = 1
 
+# Not-None sentinal for default value when None has a special meaning
+# Indicates that the last configured value should be used
+config_val = object()
```

Comment:
Nitpicky, but a global sentinel should probably be underscored and perhaps capitalized:
```suggestion
_CONFIG_VAL = object()
```

Reply:
I understand why this should be capitalized, as it falls under the global constants section in pep8.
Is the underscore because we want to hint that people should not be importing/using this value themselves?

Reply:
Yes, `_` as a hint that it's a "private" thing, so to speak.

Reply:
Thanks Ken, resolved in latest commit. I typically don't bother considering if something should be public/private any more while doing Python, but I see here that it makes sense.